source/extensions/filters/network/http_connection_manager/config.h

```diff
@@ -99,12 +100,15 @@ class InternalAddressConfig : public Http::InternalAddressConfig {
       return unix_sockets_;
     }
 
-    // TODO(snowp): Make internal subnets configurable.
+    if (!cidr_ranges_.isEmpty()) {
```

Comment:
Design alternative: if the config for this was empty, just populate `cidr_ranges_` with the rfc1918 space. I think `isInternalAddress()` predates IpList existing in the codebase.

Reply:
@ggreenway - i've added a TODO to clean up `isInternalAddress` so that `cidr_ranges_` can be populated based on RFC1918 / RFC4193 ( ipv4 vs ipv6) and follow it up with a separate PR to keep things simple.